download_action.go

```diff
@@ -115,7 +117,15 @@ func (d *DownloadAction) Run(context *DebosContext) error {
 
 	if d.Unpack == true {
 		targetdir := filename + ".d"
-		return UnpackTarArchive(filename, targetdir, d.Compression, "--no-same-owner", "--no-same-permissions")
+		err := UnpackTarArchive(filename, targetdir, d.Compression, "--no-same-owner", "--no-same-permissions")
+		if err != nil {
+			return err
+		}
+		exportSymbol = targetdir
+	}
+
+	if len(d.Name) > 0 {
```

Comment:
Name should be mandatory for download actions as you can't do anything with them unless you have the name

Reply:
done